src/transformStateWithClones.js

```diff
@@ -7,7 +8,44 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const states = [];
```

Comment:
you should change name this const, maybe: stateHistory / historyState or something like that

Reply:
> you should change name this const, maybe: stateHistory / historyState or something like that

Good point